.github/merge_rules.json

```
@@ -26,14 +36,14 @@
       ]
    },
    {
-      "name": "NVFuser",
+      "name": "NVIDIA",
```

Please keep the old name in place
```suggestion
      "name": "NVfuser",
```